### DISCUSSION TITLE
How to update from one SnapShot to the next in Ubntu?

### DISCUSSION THREAD

**Discussion by MiguelitoBeard** [stronnag/mwptools#179] [General]
Hi, New to Ubantu and trying to update to the latest SnapShot. When I double click on the .deb file the system shows me a grayed out "Installed" I know it is me not knowing what to do :) Any help?

![Screenshot from 2024-11-17 13-07-13](https://github.com/user-attachments/assets/03ec2afd-bfe0-440d-abc2-434afc1265cb)

**Comment by stronnag** [comment]
1. Download the `deb` file from github
2. `sudo apt install mwptools_24.11.15_amd64.deb`

**Comment by stronnag** [comment]
may need some sort of path specifier, e.g.
```
sudo apt install ./mwptools_24.11.15_amd64.deb
# or 
sudo apt install /tmp/mwptools_24.11.15_amd64.deb
```
depending on where the archive is.

**Reply by MiguelitoBeard** [reply]
Thanks that worked: Really looking forward to this + mLRS + INAV 8 :)
